### ISSUE TITLE
Allow multiple paths in `--commit-path`

### ISSUE THREAD

**Issue by rigor789** [conventional-changelog/conventional-changelog#909]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `git-raw-commits@2.0.0` for the project I'm working on.

We're working in a monorepo, and would like to generate changelogs for the whole repo, excluding certain folders. I was able to achieve this using the git pathspec syntax, and a small change in `git-raw-commits`.

We're now able to use the following to generate the changelog excluding a specific sub-path:
```bash
conventional-changelog -p angular -i CHANGELOG.md -s --commit-path='. :!packages/*'
```

Not sure if this would fit in the scope of this project - but I thought I'd share in case anyone runs into a similar issue.

If this is something you are willing to merge, let me know and I can prepare a PR.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/git-raw-commits/index.js b/node_modules/git-raw-commits/index.js
index 767d4a0..e76c33c 100644
--- a/node_modules/git-raw-commits/index.js
+++ b/node_modules/git-raw-commits/index.js
@@ -32,7 +32,7 @@ function getGitArgs (gitOpts) {
   // allow commits to focus on a single directory
   // this is useful for monorepos.
   if (gitOpts.path) {
-    gitArgs.push('--', gitOpts.path)
+    gitArgs.push('--', ...gitOpts.path.split(' '))
   }
 
   return gitArgs.concat(dargs(gitOpts, {
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by t-kelly** [comment]
Literally just digging into this as well! In my case, I want to filter commits that touch:
a) a specific project folder, e.g. `packages/a`
b) shared scripts across all projects, e.g. `shared`

Having the ability to specific multiple paths is exactly what's needed!

**Comment by rigor789** [comment]
@t-kelly with the above patch you would be able to pass these and I believe would work
```
--commit-path='packages/a shared'
```
(may need some trial & error to confirm)

**Comment by t-kelly** [comment]
Yep! Works like a charm. Also thank you for introducing me to patch package!!! HOW AM I ONLY JUST DISCOVERING THIS!?!

**Comment by ecnaidar** [comment]
Also looking at this in relation to https://github.com/lerna/lerna/issues/1257
Thought to generate changelogs myself but seems that it would not be possible without patching (which I would prefer to avoid)
Initially assumed that this param would accept glob pattern and was fiddling with formats so thanks for opening this 👍

**Comment by cbou** [comment]
This was fixed and merged here: https://github.com/conventional-changelog/conventional-changelog/pull/1079

You don't need to use `patch-package`. You can use `override` in `package.json`.

```
  "dependencies": {
    "conventional-changelog-core": "^7.0.0"
  },
  "overrides": {
    "conventional-changelog-core": {
      "git-raw-commits": "4.0.0"
    }
  },
```

**Comment by dangreen** [comment]
Multiple pathspecs are supported in current releases through config, for example `commits: { path: ['.', ':!packages/*'] }` or a list of directories. The git client passes the list as separate arguments after `--`, and this is regression-tested. Closing as completed.
